e2eTests/test_task_journey.py

```diff
@@ -0,0 +1,93 @@
+from time import sleep
+from threading import Thread
+
+import vcr
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
+import importlib
+import os
+import pytest
+
+from e2eTests.utils.temp_trello_boards import create_trello_board, delete_trello_board
+from todo_app import app
+from dotenv import find_dotenv, load_dotenv
+import todo_app.data.trello_client
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    with vcr.use_cassette(
+            "cassettes/e2etest_setup.yaml", filter_query_parameters=['key', 'token'], ignore_localhost=True
+    ):
+        board_id = create_trello_board("test board")
+        file_path = find_dotenv('.env')
+        load_dotenv(file_path, override=True)
```

Comment:
Is it slightly suspicious that you create the Trello board _before_ you load the environment variables? This might suggest to me that you've been running the tests through the VSCode runner, which silently loads in the `.env` file for you, but it's worth noting that this isn't the case if you run the tests from the terminal (although once handled by cassette that won't be an issue)

Reply:
Haha, you're right that there's something wrong here, but I don't think it's that (I've been running tests from either the terminal or Pycharm's test runner). 

Digging into this, Flask itself now loads environment variables from .env on load (seems like a relatively recent feature). The original reason I had this here was in case the test runner somehow carried over environment variables between running the integration tests (which use .env.test) and running the end to end tests when they're run together. Either way the create_trello_board call should be after the load_dotenv call though so I'll move it.